packages/bpk-scrim-utils/index.d.ts

```diff
@@ -18,230 +18,63 @@
 
 /// <reference types="react" />
 import withScrim from './src/withScrim';
-export { withScrim };
+import withScrimmedPortal from './src/withScrimmedPortal';
+export { withScrim, withScrimmedPortal };
 declare const _default: {
     withScrim: <P extends object>(WrappedComponent: string | import("react").ComponentType<P>) => {
-        new (props: ({
-            [rest: string]: any;
-            getApplicationElement: () => HTMLElement | null;
-            onClose?: (() => void | null) | undefined;
-            isIphone?: boolean | undefined;
-            isIpad?: boolean | undefined;
-            containerClassName?: string | undefined;
-            closeOnScrimClick?: boolean | undefined;
-        } & Omit<P, "dialogRef">) | Readonly<{
-            [rest: string]: any;
-            getApplicationElement: () => HTMLElement | null;
-            onClose?: (() => void | null) | undefined;
-            isIphone?: boolean | undefined;
-            isIpad?: boolean | undefined;
-            containerClassName?: string | undefined;
-            closeOnScrimClick?: boolean | undefined;
-        } & Omit<P, "dialogRef">>): {
+        new (props: (import("./src/withScrim").Props & Omit<P, "dialogRef">) | Readonly<import("./src/withScrim").Props & Omit<P, "dialogRef">>): {
```

Comment:
Why do we need dynamic imports here?

Reply:
Hmm, it's actually autogenerated with `tsc` not sure why it's changed this way 🤔

Reply:
Ah, okay, makes sense then